js-exercises/alternating-characters/alternatingCharacters.js

```diff
@@ -0,0 +1,17 @@
+function alternatingCharacters(arr) {
+  let result=[]
+  arr.map((value)=>{
+    let string=value.split("");
```

Comment:
Here `string` is actually an Array of characters. could have better nomenclature.